### ISSUE TITLE
Review #1875

### ISSUE THREAD

**Issue by akeller** [camunda/camunda-platform-docs#1928]
I merged docs: add Tasklist REST API docs [to master] #1875 to keep us unblocked and working toward and 8.2 docs release. 

This was merged without a review for style, grammar, etc.

This should be done in a new PR. Timing will depend on how the 8.2 docs release is progressing.

**Comment by christinaausley** [comment]
@akeller My mistake here -- this was indeed reviewed (see commits) but I had not approved yet. This can be closed 👍
